docs/2.0. APIs.md

```diff
@@ -77,34 +80,37 @@ In order to overcome shortcomings in some common libraries, the following requir
 
 #### GET and HEAD Requests
 
-* Clients performing requests using these methods SHOULD correctly handle a 301 response (moved permanently).
-* When a 301 is supported, the client MUST follow the redirect in order to retrieve the required response payload.
-* Servers implementing the APIs MUST support requests using these methods to both the trailing slash and non-trailing slash path to each resource. One of these MAY produce a 301 response causing a redirect to the other if preferred.
+- Clients performing requests using these methods SHOULD correctly handle a 301 response (moved permanently).
+- When a 301 is supported, the client MUST follow the redirect in order to retrieve the response payload.
+- Servers implementing the APIs MUST support requests using these methods to both the trailing slash and non-trailing slash path to each resource. One of these MAY produce a 301 response causing a redirect to the other if preferred.
 
 #### All Other Requests (PUT, POST, DELETE, OPTIONS etc)
 
-* Clients performing requests using these methods MUST use URLs with no trailing slash present.
-* Servers implementing the APIs MUST correctly interpret requests using these methods to paths without trailing slashes present.
-* Servers implementing the APIs MAY correctly interpret requests using these methods to paths with trailing slashes present for backward compatibility.
-* Servers SHOULD NOT respond with 3xx codes for these request types.
+- Clients performing requests using these methods MUST use URLs with no trailing slash present.
+- Servers implementing the APIs MUST correctly interpret requests using these methods to paths without trailing slashes present.
+- Servers implementing the APIs MAY correctly interpret requests using these methods to paths with trailing slashes present for backward compatibility.
+- Servers SHOULD NOT respond with 3xx codes for these request types.
 
 #### Client Behaviour
 
-When a server implementation is required to indicate an API URL via an 'href' or similar attribute, it is valid to either include a trailing slash or not provided that the listed path is accessible and abides by the policy above. In order to avoid cases of double slashes or missing slashes, clients are required to exercise caution when appending paths onto 'href' type attributes.
+When a server implementation needs to indicate an API URL via an 'href' or similar attribute, it is valid to either include a trailing slash or not provided that the listed path is accessible and follows the above rules. Clients appending paths to 'href' type attributes MUST support both cases, avoiding doubled or missing slashes.
```

Comment:
Just noting that I think this language is repeated in other specs.

Reply:
Yes. I'll be proposing similar changes for IS-05 revision.